lib/evmone/execution.cpp

```diff
@@ -19,8 +19,8 @@ evmc_result execute(AdvancedExecutionState& state, const AdvancedCodeAnalysis& a
     const auto gas_left =
         (state.status == EVMC_SUCCESS || state.status == EVMC_REVERT) ? state.gas_left : 0;
 
-    return evmc::make_result(
-        state.status, gas_left, &state.memory[state.output_offset], state.output_size);
+    return evmc::make_result(state.status, gas_left,
+        state.output_size != 0 ? &state.memory[state.output_offset] : nullptr, state.output_size);
```

Comment:
This can be fixed with `state.memory.data() + state.output_offset` because `evmc::make_result()` will not use the pointer if `output_size` is 0.

Reply:
Sure, done.